source/server/configuration_impl.cc

```diff
@@ -46,6 +47,13 @@ bool FilterChainUtility::buildFilterChain(
 void MainImpl::initialize(const envoy::config::bootstrap::v2::Bootstrap& bootstrap,
                           Instance& server,
                           Upstream::ClusterManagerFactory& cluster_manager_factory) {
+  const auto& secrets = bootstrap.static_resources().secrets();
+  ENVOY_LOG(info, "loading {} static secret(s)", secrets.size());
+  for (ssize_t i = 0; i < secrets.size(); i++) {
+    ENVOY_LOG(debug, "static secret #{}: {}", i, secrets[i].name());
+    server.secretManager().addOrUpdateSecret(server.secretManager().loadSecret(secrets[i]));
```

Comment:
these two calls can be combined into one call?

Reply:
Combined loadSecret to addOrUpdateSecret